ha/cli/command_factory.py

```diff
@@ -18,22 +18,73 @@
 import inspect
 
 from ha.cli import commands
+from ha.core.config.config_manager import ConfigManager
+from ha.cli.exec.nodeExecutor import *
```

Comment:
Such `import *` is generally discouraged because it imports literally everything. Link with more information: https://stackoverflow.com/a/2386740

Please import either required classes/functions or use `import ... as` to make shortcut if you still need entire module.

Reply:
I also think that it will fix codacy warnings about "possibly undefined" classes.

Reply:
Discussed with Ajay S, will be importing the classes individually